guest/handle/imports.go

```diff
@@ -20,5 +20,11 @@ package handle
 
 import "sigs.k8s.io/kube-scheduler-wasm-extension/guest/internal/mem"
 
+//go:wasmimport k8s.io/scheduler handle.get_waiting_pod
```

Comment:
```suggestion
//go:wasmimport k8s.io/scheduler handle.allow_waiting_pod
```

Reply:
Actually, I don't see the implementation of `allow_waiting_pod` in the host side. 
Do you want to keep the scope of this PR as only getWaitingPod? Or do you want to include that in the PR too? Either is fine to me.

Reply:
I'd like to keep the focus of this PR limited to `getWaitingPod`

Reply:
If so, please remove all the changes for `allow_waiting_pod`.